cmd/kosli/addEnvironment.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/kosli-dev/cli/internal/requests"
+	"github.com/spf13/cobra"
+)
+
+const addEnvironmentShortDesc = `Add a physical environment to a logical environment.`
+
+const addEnvironmentLongDesc = addEnvironmentShortDesc + `
+Add a physical Kosli environment to a logical Kosli environment.
+`
+
+const addEnvironmentExample = `
+# add a physical environment to a logical environment:
+kosli add environment \
```

Comment:
I realize this is probably a late comment, so feel free to ignore. Would `join` be better than `add` to avoid users confusing create and add environment commands?
`kosli join environment physical logical` , OR
`kosli join environment --physical xxx --logical xxx' , OR